modules/sandboxed-containers-create-kataconfig-resource-web-console.adoc

```diff
@@ -0,0 +1,74 @@
+//Module included in the following assemblies:
+//
+// * sandboxed_containers/deploying_sandboxed_containers.adoc
+
+[id="sandboxed-containers-create-kataconfig-resource-web-console_{context}"]
+= Creating the KataConfig custom resource in the web console
+
+You must create one `KataConfig` custom resource (CR) to enable installing the Kata runtime on your cluster nodes.
+
+.Prerequisites
+
+* You have {product-title} {product-version} installed on your cluster.
+* You have access to the cluster as a user with the `cluster-admin` role.
+* You have installed the {sandboxed-containers-operator}.
+
+[NOTE]
+====
+The Kata runtime is installed on all worker nodes by default. To install the Kata runtime only on selected nodes, you must label the nodes first, and then create the `KataConfig` CR and include the label in the YAML file.
+====
+
+.Procedure
+
+. From the *Administrator* perspective in the web console, navigate to *Operators* → *Installed Operators*.
+
+. Select the {sandboxed-containers-operator} from the list of operators.
+
+. In the *KataConfig* tab, click *Create KataConfig*.
+
+. In the *Create KataConfig* page, select to configure the `KataConfig` CR via *YAML view*.
+
+. Copy and paste the following manifest into the YAML view:
+
++
+[source,yaml]
+----
+apiVersion: kataconfiguration.openshift.io/v1
+kind: KataConfig
+metadata:
+  name: cluster-kataconfig
+----
++
+If you want to install the Kata runtime only on selected nodes, include the label in the manifest:
+
++
+[source,yaml]
+----
+ apiVersion: kataconfiguration.openshift.io/v1
+ kind: KataConfig
+ metadata:
+   name: cluster-kataconfig
+ spec:
+   kataConfigPoolSelector:
+     matchLabels:
+     <label_key>: '<label_value>'
+----
+
+. Click *Create*.
+
+The new KataConfig CR is created and begins to install the kata runtime on the worker nodes.
```

Comment:
Backticks are missing (`KataConfig`) and "kata runtime" is not capitalized.

Reply:
Done.